Application/Platform/Gatekeeper/Authorization/Consumer/Service/Data.php

```diff
@@ -86,6 +84,52 @@ public function getConsumerById($Id)
         return $this->getCachedEntityById(__METHOD__, $this->getConnection()->getEntityManager(), 'TblConsumer', $Id);
     }
 
+    /**
+     * @param integer $Id
+     *
+     * @return bool|TblConsumer
+     */
+    public function getConsumerLoginById($Id)
+    {
+
+        return $this->getCachedEntityById(__METHOD__, $this->getConnection()->getEntityManager(), 'TblConsumerLogin', $Id);
+    }
+
+    /**
+     * @param TblConsumer $tblConsumer
+     *
+     * @return bool|TblConsumerLogin
+     */
+    public function getConsumerLoginByConsumer(TblConsumer $tblConsumer)
+    {
+
+        return $this->getCachedEntityBy(__Method__, $this->getConnection()->getEntityManager(), 'TblConsumerLogin',
+            array(TblConsumerLogin::ATTR_TBL_CONSUMER => $tblConsumer->getId())
+        );
+    }
+
+    /**
+     * @param string $SystemName
+     *
+     * @return bool|TblConsumerLogin[]
+     */
+    public function getConsumerLoginBySystemName($SystemName = '')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$SystemName" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)